R/tags.R

```diff
@@ -698,7 +713,35 @@ findDependencies <- function(tags, tagify = TRUE) {
     }
   }
   childDeps <- unlist(lapply(children, findDependencies, tagify = FALSE), recursive = FALSE)
-  c(childDeps, if (!is.null(dep)) dep)
+  c(childDeps, if (!is.null(deps)) deps)
```

Comment:
The `!is.null` check can actually be removed. Because when an `if` statement isn't true, it the whole thing evaluates to `NULL`.

```
print(if (F) "hello")
#> NULL
```